### PR TITLE
[O11y][ActiveMQ] Remove forwarded tag from metrics data streams

### PR DESCRIPTION
**Pull request by harnish-elastic**
- Bug

## What does this PR do?

- This PR remove forwarded tag from the metricbeat-based data streams.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #7635

### CODE REVIEW

File: packages/activemq/data_stream/topic/manifest.yml [hunk 1]
```diff
@@ -17,9 +17,8 @@ streams:
         required: false
         show_user: false
         default:
-          - forwarded
           - activemq-topic
     title: ActiveMQ topic
     description: Collect ActiveMQ topic metrics
 elasticsearch:
-  index_mode: "time_series"
```
Comment (ishleenk17) [inline packages/activemq/data_stream/topic/manifest.yml:25]: Why is this getting changed ?

Comment (harnish-elastic) [reply packages/activemq/data_stream/topic/manifest.yml:25]: Due to the line shift, changes are being displayed.

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Pull request merged**